bedrock/products/models.py

```diff
@@ -266,9 +266,7 @@ def __init__(self, *args, **kwargs):
 
     def get_context(self, request, *args, **kwargs):
         context = super().get_context(request, *args, **kwargs)
-        context["articlepages"] = (
-            MonitorArticlePage.objects.live().public().order_by("-first_published_at")
-        )  # live() and public() are wagtail filters
+        context["articlepages"] = MonitorArticlePage.objects.live().public().in_menu().order_by("path")
```

Comment:
I'm not 100% sure about overloading the show_in_site_menus boolean here. While we aren't likely to use it because we control our nav and footer very tightly, it is kind of going against the Wagtail idiom of what that field is for, and may cause confusion for users

Instead, what do you think about us adding a `BooleanField` called `show_in_index_pages` (default: `True`) in either `AbstractCMSPage` or just `MonitorArticlePage`, or in a new `ArticlePageMixin` class, which we use in `MonitorArticlePage`, and then filter on that value?

Reply:
Hm. I must have misunderstood what you meant when you said Wagtail had a built in feature for weighting items. Do you remember what your recommendation was for sorting the order of articles listed on the index page?